Solutions/Web Session Essentials/Hunting Queries/ExcessiveForbiddenRequestsDetected.yaml

```diff
@@ -0,0 +1,48 @@
+id: 43c2832e-6c01-4dc1-bd9e-bc3f330c2b31
+name: Excessive number of forbidden requests detected (ASIM Web Session)
+description: |
+  'This rule detects abnormal number of 403 errors from clients. HTTP 403 is returned when the client is not permitted access to the resource despite providing authentication in case such as when authenticated account not having sufficient permissions'
+tags:
+  - Schema: WebSession
+    SchemaVersion: 0.2.6
+requiredDataConnectors: []
+tactics:
+  - Persistence
+  - CredentialAccess
+relevantTechniques:
+  - T1110
+  - T1556
+query: |
+  // Please refer this for more details: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status
+  let threshold=100; // Update threshold as per your environment
+  let lookBack = 1d;
+  let ErrorCode=dynamic([403]);
+  _Im_WebSession(starttime=ago(lookBack))
```

Comment:
Can we can use the in-built filters for ErrorCode filter?

Reply:
there seems to be some issue with filter. not using filter currently to avoid rule malfunction. Will resolve filter separately.